Tests/DummyEntities.php

```diff
@@ -46,6 +46,11 @@ class DummyUser implements PreferencesAwareUserInterface
      */
     public $groups;
 
+    /**
+     * @Orm\Column(type="integer", nullable=true)
+     */
+    public $price = 0;
```

Comment:
Why this has been added? I don't see it being checked anywhere in the tests.

Reply:
https://github.com/sergeil/SliExtJsIntegrationBundle/blob/master/Tests/QueryBuilder/DqlCompilerTest.php#L63